manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml

```diff
@@ -25,6 +25,7 @@ spec:
     disableKServe: false
     disableKServeAuth: false
     disableKServeMetrics: false
+    disableKServeRaw: true
```

Comment:
We should not be adding feature flags to our custom resource definition or config any more until it is ready for release.
The rest of the code can stay and you can toggle the feature flag using the dev feature flags.

Reply:
Thanks, I removed it, should i remove it from `manifests/common/crd/odhdashboardconfigs.opendatahub.io.crd.yaml` as well?

Reply:
@emilys314 good question. I'm inclined to say remove it. 

The reason I say this is because in general there's potential for a feature flag to change over the course of development and we don't want to release a flag early that will change or give the customer an ability to disable and then when we do release the feature, they won't get the new default.